app/scenes/Settings/Security.tsx

```diff
@@ -117,6 +173,28 @@ function Security() {
         />
       </SettingRow>
 
+      <SettingRow
+        label={t("Allow authorized signups")}
+        name="allowSignups"
+        description={
+          <Trans
+            defaults="Allow authorized <em>{{ authMethodSnippet }}</em> users to create new accounts without first receiving an invite"
```

Comment:
```suggestion
            defaults="Allow authorized {{ authMethodSnippet }} users to create new accounts without first receiving an invite"
```

Reply:
I would rather keep the <em>s around authMethodSnippets